src/main/scala/io/citrine/lolo/trees/Nodes.scala

```diff
@@ -43,12 +79,18 @@ trait ModelNode[T <: PredictionResult[Any]] extends Serializable {
   * @param split to decide which branch to take
   * @param left  branch node
   * @param right branch node
+  * @param numFeatures number of features, used for Shapley computation
+  * @param outputDimension dimension of model output, used for Shapley computation
```

Comment:
Its not clear to me what this number is.  Can you give an example, like "1 for single-task regression" or "equal to the number of categories in classification"?